internal/cluster/transition.go

```diff
@@ -30,6 +30,10 @@ import (
 )
 
 var resetLogsField = []interface{}{"logs_info", "", "controller_logs_started_at", strfmt.DateTime(time.Time{}), "controller_logs_collected_at", strfmt.DateTime(time.Time{})}
+var resetProgressFields = []interface{}{"progress_finalizing_stage_percentage", 0, "progress_installing_stage_percentage", 0,
+	"progress_preparing_for_installation_stage_percentage", 0, "progress_total_percentage", 0}
```

Comment:
What about resetting the host progress? This will only reset the cluster progress.